app/models/lookup.js

```diff
@@ -0,0 +1,9 @@
+import Model from "ember-data/model";
```

Comment:
@abhinav098 @akshaygwd  I'm probably a bit late on that end. But any possibilities of renaming the table to `form-lookup` (although that belongs more on the api PR than here).

If it's a lot of effort then carry on as it is, I just think it's more descriptive and understandable for anyone that might join the project later on 😄